migrations/20240130125545-add-expense-payment-method.js

```diff
@@ -0,0 +1,96 @@
+    `);
+    console.info(metadata.rowCount, 'expenses updated with stripe virtual card payment method');
+
+    [, metadata] = await queryInterface.sequelize.query(`
+      WITH paypalpayouts AS (SELECT DISTINCT e."HostCollectiveId" FROM "Expenses" e WHERE e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{payout_batch_id}' IS NOT NULL)
+      INSERT INTO "PaymentMethods" ("service", "type", "CollectiveId", "saved", "data")
+      SELECT 'paypal' as "service", 'payout' as "type", "HostCollectiveId" as "CollectiveId", False as "saved", '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB as "data" FROM paypalpayouts ON CONFLICT DO NOTHING;
+    `);
+    console.info(metadata.rowCount, 'paypal payout payment methods created');
+    [, metadata] = await queryInterface.sequelize.query(`
+      UPDATE "Expenses" as e
+      SET "PaymentMethodId" = pm.id, "data" = '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB || e.data
+      FROM "PaymentMethods" pm
+      WHERE
+        e."deletedAt" IS NULL AND e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{payout_batch_id}' IS NOT NULL
+        AND pm."service" = 'paypal' AND pm.type = 'payout' AND pm."CollectiveId" = e."HostCollectiveId"
+      RETURNING e.*;
+    `);
+    console.info(metadata.rowCount, 'expenses updated with paypal payout payment method');
+
+    [, metadata] = await queryInterface.sequelize.query(`
+      WITH paypaladaptive AS (SELECT DISTINCT e."HostCollectiveId" FROM "Expenses" e INNER JOIN "PayoutMethods" as po ON po.id = e."PayoutMethodId" WHERE e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{payout_batch_id}' IS NULL AND e.data#>>'{quote,id}' IS NULL AND e.data#>>'{transaction,id}' IS NULL AND po.type = 'PAYPAL')
+      INSERT INTO "PaymentMethods" ("service", "type", "CollectiveId", "saved", "data")
+      SELECT 'paypal' as "service", 'adaptive' as "type", "HostCollectiveId" as "CollectiveId", False as "saved", '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB as "data" FROM paypaladaptive ON CONFLICT DO NOTHING;
+    `);
+    console.info(metadata.rowCount, 'paypal adaptive payment methods created');
+    [, metadata] = await queryInterface.sequelize.query(`
+      UPDATE "Expenses" as e
+      SET "PaymentMethodId" = pm.id, "data" = '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB || e.data
+      FROM "PaymentMethods" pm, "PayoutMethods" po
+      WHERE
+        e."deletedAt" IS NULL AND e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{payout_batch_id}' IS NULL AND e.data#>>'{quote,id}' IS NULL AND e.data#>>'{transaction,id}' IS NULL
+        AND po.id = e."PayoutMethodId" AND po.type = 'PAYPAL'
+        AND pm."service" = 'paypal' AND pm.type = 'adaptive' AND pm."CollectiveId" = e."HostCollectiveId"
+      RETURNING e.*;
+    `);
+    console.info(metadata.rowCount, 'expenses updated with paypal adaptive payment method');
+  },
+
```

Comment:
Maybe missing is Payout Method ACCOUNT_BALANCE -> service: opencollective, type: collective